### PR TITLE
A22AB1-22 created route to get a single inventory item by providing i…

### PR DESCRIPTION
**Pull request by ghost**
route created to get a single inventory item and also created a if check to make sure the id is sent to the api endpoint

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const cors = require("cors");
 const dotenv = require("dotenv");
 dotenv.config();
 const getAllWarehouses = require("./routes/getallwarehouses/getAllWarehouses");
```
Comment (kevinlinpt) [inline index.js:12]: should be camelcase for 'getallwarehouses' here?

File: routes/GetInventoryItem/GetInventoryItem.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const express = require('express')
+const router = express.Router()
+const inventoryItems = require('../../data/inventories.json')
+
+router.get('/',function(req,res){
+  if(req.body.id){
+
```
Comment (kevinlinpt) [inline routes/GetInventoryItem/GetInventoryItem.js:7]: nitpick: remove additional row

**Review by kevinlinpt — APPROVED** [review]
2 minor nitpicks, otherwise ready to merge

File: index.js (hunk 1 above)
Comment (ghost) [reply index.js:12]: that's caitlin's I will let her know

**Pull request merged**